marimbabot_planning/include/marimbabot_planning/planning.h

```diff
@@ -15,7 +15,7 @@
 #include <tf2_ros/transform_listener.h>
 #include <tf2/LinearMath/Quaternion.h>
 #include <trajectory_msgs/JointTrajectoryPoint.h>
-
+#include <list>
```

Comment:
Please only use a list if you perform random insertions or deletions of elements at other places than the end. In most other cases (like this one) a `std::vector` is the data structure you want to use in cpp.